src/portable/portablectl.c

```diff
@@ -508,7 +510,82 @@ static int maybe_enable_start(sd_bus *bus, sd_bus_message *reply) {
 
                 if (STR_IN_SET(type, "symlink", "copy") && ENDSWITH_SET(path, ".service", ".target", ".socket")) {
```

Comment:
hmm, the unit type list seems out of date, .path and .timer is missing there, iiuc

Reply:
added and refactored